install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -318,8 +318,13 @@ exported volume on the NFS server should conform to the following conditions:
 - Each export must be:
 +
 ----
-/<example_fs> *(rw,root_squash)
+/<example_fs> *(rw,root_squash,sync,no_wdelay)
 ----
++
+Where the `sync` option causes server to commit all the changes to the
+underlying storage before sending a reply. And `no_wdelay` stops the server
+from delaying the writes. They proved to be necessary for correct functioning
```

Comment:
`no_wdelay` works only with `sync`. But `sync` is the default. So it should be fine to omit it. I'll do the change.